src/openforms/js/lang/nl.json

```diff
@@ -755,7 +770,7 @@
     "originalDefault": "Component"
   },
   "MXKqVO": {
-    "defaultMessage": "Below you can specify a custom template for the configuration email. If not specified, the globally configured template will be used.",
+    "defaultMessage": "Je kan hieronder een formulierspecifiek sjabloon opstellen voor de bevestigingsmail. Indien leeg, dan worden de algemene instellingen gebruikt.",
```

Comment:
"Je kan" is informal - if all language in the project is formal then use "Je kunt".

Reply:
I've updated it to not address the reader at all, but rather give a description which is more consistent with other texts (no precedent for "U kunt", "U kan", "Je kan" or "Je kunt" at all).